src/App.css

```diff
@@ -30,4 +30,5 @@ input[type="text"] {
   grid-template-columns: 1fr 1fr 1fr;
   grid-template-rows: 1fr 1fr 1fr;
   gap: 15px 15px;
+  margin-right: 31px;
```

Comment:
This is not the right way to do it. Why adding "random" margin on the right side?
Check the page/structure: it is a `<ul>` so it has a default left padding. The better way to center the elements is:
```
.boxForItems{ padding: 0}
.foodListItems {margin:auto}
```
(just adding that to the existing styles)

Reply:
Okay, I will work on this.
Thank you for your guidance.